torch/distributions/multivariate_normal.py

```diff
@@ -159,8 +159,10 @@ def covariance_matrix(self):
     @lazy_property
     def precision_matrix(self):
         # TODO: use `torch.potri` on `scale_tril` once a backwards pass is implemented.
-        flat_conv = self.covariance_matrix.reshape((-1,) + self._event_shape * 2)
-        flat_precision = torch.stack([C.inverse() for C in flat_conv], 0)
+        scale_tril_inv = _batch_inverse(self.scale_tril)
+        flat_scale_tril_inv = self.scale_tril.reshape((-1,) + self._event_shape * 2)
```

Comment:
I think this is the error. I believe it is supposed to be `self.scale_tril_inv`.

Reply:
Fixed in #6172 .